Sources/App/Controllers/PackageController.swift

```diff
@@ -0,0 +1,18 @@
+import Fluent
+import Plot
+import Vapor
+
+struct PackageController {
+
+    func index(req: Request) throws -> EventLoopFuture<Response> {
+        // No such thing as a full index of packages.
+        return req.eventLoop.future(req.redirect(to: "/"))
```

Comment:
Oh nice, I had no idea there's an `eventLoop.future` :D

Reply:
Well, when you've been doing Vapor development as long as I have, you learn a few things... 😂 Orrrrr... Google is *awesome* 🤷‍♂️

Actually I learned something cool about redirects when figuring this out. My initial feeling was just to make this a 302, but a 303 enforces the redirect to be a GET regardless of the method used for the initial request. It's not actually changing anything here as `index` is always a GET in this case, but it's good to know.